### PR TITLE
Let there be art! Rainbow Crayon Buff

### PR DESCRIPTION
**Pull request by Avalon-Proto**
## About the PR
Simply changed rainbow crayons to be nigh infinite

## Why / Balance
Gives clowns or anyone else who gets their hands on one ot draw full murals of art rather than just a few scribbles here or there

## Technical details
Changed 1 value, and made rainbow crayons near impossible to eat as well

## Requirements

- [x] I have tested all added content and changes.
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

🆑 
- tweak: Artists rejoice! Rainbow crayons are now nigh infinite!

### CODE REVIEW

**Comment by Radezolid** [comment]
Lmao, I was about to do this after seeing the discord image.

**Comment by dvir001** [comment]
Consider:
https://github.com/new-frontiers-14/frontier-station-14/pull/2157

**Comment by Avalon-Proto** [comment]
> Consider: [new-frontiers-14/frontier-station-14#2157](https://github.com/new-frontiers-14/frontier-station-14/pull/2157)

Honestly this is way better than my back job and I'd love to have it but I dunno how to port things properly :imcry:

File: Resources/Prototypes/Entities/Objects/Fun/crayons.yml [hunk 1]
```diff
@@ -88,13 +88,21 @@
...
     - Write
     - Crayon
     - Recyclable
     - Trash
+  - type: SolutionContainerManager # Delta V - So they can't accidentally eat it
+    solutions:
+      food:
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 99998
```
Comment (Lyndomen) [inline Resources/Prototypes/Entities/Objects/Fun/crayons.yml:103]: This value seems absurd. Even if the intent is to disallow them eating it, making it the greatest source of food in the game is not that. Just disable eating crayons if that's the idea, or make it an option that can only be done via verb menu.

File: Resources/Prototypes/Entities/Objects/Fun/crayons.yml [hunk 2]
```diff
@@ -88,13 +88,21 @@
   - type: Crayon
     color: Red
     selectableColor: true
-    capacity: 30
+    capacity: 1000000 # Delta V - Let there be art!
```
Comment (Lyndomen) [inline Resources/Prototypes/Entities/Objects/Fun/crayons.yml:91]: I highly suggest making this a more reasonable value. The functional difference between 1,0000,000 and something like 100 is not that large, but prevents abuse. Also, crayons do get used up irl - maybe add in a new research for bluespace crayons that have infinite uses if you really want that.

Comment (Avalon-Proto) [reply Resources/Prototypes/Entities/Objects/Fun/crayons.yml:91]: I am being honest- I will probably try and figure out how to port later and instead add the magic crayon PR that was thrown at me in the above comments

**Pull request closed without merging**